centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -2914,8 +2914,8 @@ msgstr "Activités métiers"
 msgid "Business Views"
 msgstr "Vues métiers"
 
-# centreon-web/www/install/menu_translation.php:121
-msgid "Monitoring Connectors Manager"
+#: centreon-web/www/install/menu_translation.php:121
+msgid "Monitoring Connector Manager"
```

Comment:
All references to “Monitoring Connectors Manager” must be replaced in the code, and it must be checked that there is no duplication with module translations.

Reply:
We talked about this during the planing I think. Another ticket says this menu must be renamed "Monitoring connectors" only - there is no point in fixing this translation then.